activejob/test/support/integration/adapters/sneakers.rb

```diff
@@ -84,7 +84,7 @@ def bunny_publisher
     end
 
     def bunny_queue
-      @queue ||= bunny_publisher.exchange.channel.queue "integration_tests", durable: true
+      @queue ||= bunny_publisher.send(:exchange).channel.queue "integration_tests", durable: true
```

Comment:
Yes. exchange moved to private api in this commit https://github.com/jondot/sneakers/commit/c331d118f5b2f4cdb5c5f7a5d06cadf8cbc5f40e